src/components/todo/TodoList.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { connect } from 'react-redux';
+import TodoListItem from "./TodoListItem";
+import NewTodoForm from "./NewTodoForm";
+import { removeTodo, markTodoAsCompleted } from "./actions";
+
+import  './TodoList.css';
+
+const TodoList = ({todos = [], onRemovePress, onCompletedPress}) => {
+    return (
+        <div className="list-wrapper">
+            <NewTodoForm />
+            {todos.map((todo, index) => <TodoListItem 
+                todo={todo} 
+                key={index}
+                onRemovePress={onRemovePress}
+                onCompletedPress={onCompletedPress} />
+                )}
+        </div>
+    )
+}
+
+const mapStateToProps = state => ({
+    todos: state.todos,
+});
+
+const mapDispatchToProps = dispatch => ({
+    onRemovePress: text => dispatch(removeTodo(text)),
+    onCompletedPress: text => dispatch(markTodoAsCompleted(text)),
+});
+export default connect(mapStateToProps, mapDispatchToProps)(TodoList);
```

Comment:
using connect like this is kinda a hold way of doing this. I would look at using redux hooks